crates/orchestrator/tests/jobs/da_job/mod.rs

```diff
@@ -0,0 +1,137 @@
+use rstest::*;
+use starknet::core::types::StateUpdate;
+
+use std::fs::read_to_string;
+// use std::io::{self, BufRead};
+use serde_json::Error;
+use std::sync::Arc;
+use mockall::predicate::*;
+
+use starknet_core::types::{BlockId, FieldElement, MaybePendingStateUpdate};
+use crate::common::{
+    get_or_init_config,
+    default_job_item,
+};
+
+use da_client_interface::MockDaClient;
+
+use orchestrator::{
+    config::{MockConfig, Config, MyStarknetProvider, MockMyStarknetProvider},
+    jobs::{
+        da_job::DaJob,
+        types::{JobItem, JobType},
+        Job
+    },
+};
+
+#[fixture]
+fn state_update() -> StateUpdate {
+    let state_update_path = "test-utils/stateUpdate.json".to_owned();
+    let contents = read_to_string(state_update_path).expect("Couldn't find or load that file.");
+
+    let v: Result<StateUpdate, Error> = serde_json::from_str(&contents.as_str());
+
+    let state_update: StateUpdate = match v {
+        Ok(state_update) => state_update,
+        Err(e) => panic!("Couldn't parse the JSON file: {}", e),
+    };
+
+    // let blob_data = state_update_to_blob_data(630872, state_update);
+    state_update
+}
+
+#[fixture]
+fn blob_data() -> Vec<FieldElement> {
+    vec![FieldElement::ONE, FieldElement::ZERO]
+}
+
+#[rstest]
+#[tokio::test]
+async fn test_create_job(
+    #[future] #[from(get_or_init_config)] config: &'static dyn Config,
+) {
+    let config = config.await;
+    let job = DaJob.create_job(config, String::from("0")).await;
+    assert!(job.is_ok());
+
+    let job = job.unwrap();
+    
+    let job_type = job.job_type;
+    assert_eq!(job_type, JobType::DataSubmission);
+    assert_eq!(job.metadata.values().len(), 0, "metadata should be empty");
```

Comment:
not a valid test point, maybe other job types will add meta data. better to just test the 

1. type is DataSubmission
2. check if job_id is present
3. status is created
4. version is 0